src/main/scala/edu/ie3/mobsim/io/probabilities/FirstDepartureOfDay.scala

```diff
@@ -24,21 +29,27 @@ final case class FirstDepartureOfDay(
     */
   def sample(time: ZonedDateTime): ZonedDateTime = {
 
+    @tailrec
+    def sampleNextDayWithNewTrip(time: ZonedDateTime): ZonedDateTime = {
+      if (random.nextDouble() <= averageCarUsage) time.plusDays(1)
+      else sampleNextDayWithNewTrip(time.plusDays(1))
+    }
```

Comment:
Do we have to limit this to the maximum days of simulation? Since it could happend that the random is always above the averageCarUsage we would be stucked here, or?